package.json

```diff
@@ -7,7 +7,7 @@
     "start:parrot": "concurrently --kill-others 'npm run start -- --env.mock' 'parrot-server --port 3001 --scenarios ./parrot.scenarios.js'",
     "test:unit": "jest --testPathIgnorePatterns browser",
     "test:browser": "jest browser --preset jest-puppeteer-preset",
-    "test": "npm run test:unit && npm run test:browser"
+    "test": "jest"
```

Comment:
Should the `test:unit` and `test:browser` scripts be removed now?

Reply:
adding to this should we move the jest config to a config file?

Reply:
@anescobar1991 - updated in 60ab26c

Reply:
@nellyk - originally I had done that, @JamesSingleton requested moving it back (https://github.com/americanexpress/parrot-jest-image-snapshot-example/pull/1#discussion_r362033418). I think the jest config in the `package.json` is okay for testing config reqs currently.